rke/structure_rke_cluster.go

```diff
@@ -505,3 +513,24 @@ func expandRKEClusterFlag(in *schema.ResourceData, clusterFilePath string) clust
 
 	return obj
 }
+
+func k8sVersionRequiresCri(kuberenetsVersion string) bool {
+	metadata.InitMetadata(context.Background())
+	versions := make([]*version.Version, 0, len(metadata.K8sVersionToRKESystemImages))
+	for k := range metadata.K8sVersionToRKESystemImages {
+		v, _ := version.NewVersion(k)
+		versions = append(versions, v)
+	}
+
+	sort.Sort(sort.Reverse(version.Collection(versions)))
+	kuberenetsVersion = kuberenetsVersion[1:]
```

Comment:
Why are we we slicing the first element from the `kubernetesVersion`? Won't that mean the for loop will always break out?

Reply:
I'm getting the version from the metadata of the RKE package. The versions there don't have the "v" like we require on the terraforming provider. 
That's  why I'm cutting the 1st element of our version.

Reply:
Ok. Can you add an inline comment specifying that?